lib/pro/set-status.js

```diff
@@ -2,6 +2,7 @@ module.exports = setStatusPro
 
 const getAppConfig = require('../app-config')
 const PRO_PLAN_FOR_FREE = require('../../pro-plan-for-free')
+const emojiText = require('emoji-text')
```

Comment:
`emoji-text` is only a few lines of code and no longer maintained. It uses the `wemoji` package which tries to be a drop-in relpacement for `gemoji`. [`gemoji`](https://github.com/wooorm/gemoji) sounds like what we want?

> Gemoji (GitHub Emoji) maps Gemoji short-codes and unicode emoji to objects containing info on their respective category, description, names, and tags